setup.sh

```diff
@@ -1,29 +1,47 @@
 #!/bin/sh
+# Check if nix is already installed
+if ! which nix > /dev/null 2>&1; then
```

Comment:
Is there a reason for sending STDER to `/dev/null`?

Reply:
Nope, I will just pipe stdout.